content/blog/2016-10-29-unix-tools-for-windows/index.md

```diff
@@ -9,7 +9,9 @@ categories:
     - GMT工具
 tags:
     - Linux
-    - Windows
+    - Windows    
+redirect: https://docs.gmt-china.org/latest/install/windows/#id1
```

Comment:
Need to be updated if https://github.com/gmt-china/GMT_docs/pull/512 is accepted.